### ISSUE TITLE
Missing app

### ISSUE THREAD

**Issue by Shoaibans** [stores-cedcommerce/Stephanie-Goldman-Speed#4]
The accessibility app is missing from the preview.
Preview:
![image](https://user-images.githubusercontent.com/97726944/169788987-7627842f-de15-4bc8-9bf1-36656265ef7c.png)

Live:
![image](https://user-images.githubusercontent.com/97726944/169789047-1e1890cf-b203-46cb-96d7-dc90d975e174.png)

**Comment by devabulabbas** [comment]
@Shoaibans 
![image](https://user-images.githubusercontent.com/100209490/170270343-37d6871e-de7e-4774-9449-9f04fe43daa6.png)
resolved
